constants/fluid.py

```diff
@@ -1,2 +1,4 @@
-vaultResolver = "0x56ddF84B2c94BF3361862FcEdB704C382dc4cd32"
-vaultFactory  = "0x324c5Dc1fC42c7a4D43d92df1eBA58a54d13Bf2d"
\ No newline at end of file
+vaultResolver = "0x49290f778faAD125f2FBCDE6F09600e73bf4bBd9"
+vaultPositionResolver = "0x3E3dae4F30347782089d398D462546eb5276801C"
+vaultFactory  = "0x324c5Dc1fC42c7a4D43d92df1eBA58a54d13Bf2d"
+susde = "0x9D39A5DE30e57443BfF2A8307A4256c8797A3497"
```

Comment:
we can use Token.SUSDE everywhere?

Reply:
@0xprinc I mean completely remove suSDe here and use the already defined `Token.SUSDE`

Reply:
using it gives unexpected output.
imo it represents only string "SUSDE" because

```
    print(Token.SUSDE == "0x9D39A5DE30e57443BfF2A8307A4256c8797A3497")
    print(Token.SUSDE.name == "0x9D39A5DE30e57443BfF2A8307A4256c8797A3497")
    print(Token.SUSDE.value == "0x9D39A5DE30e57443BfF2A8307A4256c8797A3497")
    
    print(Token.SUSDE == "0x9d39a5de30e57443bff2a8307a4256c8797a3497")
    print(Token.SUSDE.name == "0x9d39a5de30e57443bff2a8307a4256c8797a3497")
    print(Token.SUSDE.value == "0x9d39a5de30e57443bff2a8307a4256c8797a3497")
```

has this following output

```
False
False
False
False
False
False
Token.SUSDE
SUSDE
sUSDe
```

Reply:
oh ok!